### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by jbgwu**
Edited for punctuation and a broken link.

### CODE REVIEW

File: src/content/developers/tutorials/erc-721-vyper-annotated-code/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ In contrast to that, ERC-721 tokens are designed for assets that are similar but
 cartoons](https://www.cryptokitties.co/)
 or titles to different pieces of real estate.
 
-In this article we will analyze [Ryuya Nakamura's ERC-721 contract](https://github.com/vyperlang/vyper/blob/master/examples/tokens/ERC721.vy).
+In this article we will analyze [Ryuya Nakamura's ERC-721 contract](https://github.com/vyperlang/vyper/blob/master/vyper/builtin_interfaces/ERC721.py).
```
Comment (wackerow) [inline src/content/developers/tutorials/erc-721-vyper-annotated-code/index.md:20]: Little confused, the original link is working for me and appears to be the contract being referenced. Could we revert this? Or am I missing something?

**Review by wackerow — CHANGES_REQUESTED** [review]
@jbgwu Thanks for posting =) Left a question about the link you swapped out, otherwise this looks good 👍🏻

**Comment by minimalsm** [comment]
Hey @jbgwu ✌️ Thanks for the PR. Agree with Paul that the original reference to the vyper file was the correct one so I've reverted that (feel free to comment if you think we've made a mistake). Appreciate the improvements you've made.

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!.

**Comment by minimalsm** [comment]
@all-contributors please add @jbgwu for docs

**Pull request merged**